src/pages/cplusplus/index.md

```diff
@@ -92,7 +92,7 @@ _You have finished coding your first C++ program and have understood most of the
  
  **Feel free to ask any questions on FreeCodeCamp's GitHub page or [FreeCodeCamp's Forum.](https://forum.freecodecamp.org/)**
 
- <a href='https://repl.it/L4k3' target='_blank' rel='nofollow'>Try it yourself ! :) </a>
+ <a href='https://repl.it/L4k3' target='_blank' rel='nofollow'>Try it yourself ! :)</a>

```

Comment:
Hi @Xiryl 

Can you please change this to use Markdown formatting? This is using HTML anchor tags which isn't correct.